### PR TITLE
created new terms, fixes ##760

### PR DESCRIPTION
**Pull request by pgaudet**

### CODE REVIEW

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6284,6 +6285,26 @@ AnnotationAssertion(rdfs:label obo:RO_0012012 "indirectly regulates"@en)
 SubObjectPropertyOf(obo:RO_0012012 obo:RO_0002211)
 SubObjectPropertyOf(obo:RO_0012012 obo:RO_0012011)
 
+# Object Property: obo:RO_0012013 (positively regulates in another organism)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0012013 "x 'positively regulates in other organism' y if and only if: (x is the realization of a function to increase the frequency, rate or extent of y) AND (the agents of x are produced by organism o1 and the agents of y are produced by organism o2).")
+AnnotationAssertion(obo:RO_0004049 obo:RO_0012013 obo:RO_0002010)
+AnnotationAssertion(oboInOwl:created_by obo:RO_0012013 "pg")
```
[the remark this answers is not in the document]
Comment (pgaudet) [reply src/ontology/ro-edit.owl:6292]: Thanks! Is this supposed to be a 'literal'?

**Comment by pgaudet** [comment]
Hi @anitacaron I am not sure these changes are right; there are many lines changed in the diff, that are unrelated to the change I just made. 

Thanks, Pascale

File: src/ontology/ro-edit.owl (hunk 1 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply src/ontology/ro-edit.owl]: No, you did the right way in the new commit. Thank you!

**Comment by pgaudet** [comment]
I cannot update right away - usually all GO curators update at once, and we haven't checked that the new protégé is not causing any issues yet

**Comment by pgaudet** [comment]
HI @anitacaron 
Let me know if this looks better. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Hi @anitacaron 

Is this new PR OK? 

Thanks, Pascale

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -42,6 +42,9 @@ Annotation(foaf:homepage " https://github.com/oborel/obo-relations/"^^xsd:anyURI
 Declaration(Class(obo:BFO_0000019))
 Declaration(Class(obo:BFO_0000031))
 Declaration(Class(obo:CHEBI_50906))
+Declaration(Class(obo:CL_0000000))
+Declaration(Class(obo:CL_0000101))
+Declaration(Class(obo:CL_0000540))
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:47]: are you referring to these terms? Could you move them to `src/ontology/imports/cl_terms.txt`, please?

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by anitacaron** [comment]
Someone else needs to review the new term. I can only review it technically.

**Comment by pgaudet** [comment]
OK, thanks 

@cmungall Could you please review these?

**Review by cmungall — DISMISSED** [review]

**Comment by pgaudet** [comment]
@anitacaron Is there still anything blocking to merge this? 

Thanks, Pascale

File: src/ontology/ro-edit.owl (hunk 2 above)
Comment (anitacaron) [reply src/ontology/ro-edit.owl:47]: @pgaudet ping

Comment (pgaudet) [reply src/ontology/ro-edit.owl]: Thanks! I tried to fix this (it was already in the CL import file, so that hasn't changed)

Comment (pgaudet) [reply src/ontology/ro-edit.owl]: Is this now OK to merge?

**Review by cmungall — APPROVED** [review]

File: src/ontology/ro-edit.owl (hunk 2 above)
Comment (cmungall) [reply src/ontology/ro-edit.owl]: yes

**Review by anitacaron — APPROVED** [review]

**Pull request merged**